torch/utils/inference/run.py

```diff
@@ -0,0 +1,569 @@
+r"""
+This is a script for launching PyTorch inference on Intel(R) Xeon(R) Scalable Processors with optimal configurations.
+Single instance inference, multi-instance inference are enabled.
+
+To get the peak performance on Intel(R) Xeon(R) Scalable Processors, the script optimizes the configuration of thread and memory
+management. For thread management, the script configures thread affinity and the preload of Intel OMP library.
+For memory management, it configures NUMA binding and preload optimized memory allocation library (e.g. tcmalloc, jemalloc).
+
+**How to use this module:**
+
+*** Single instance inference ***
+
+1. Run single-instance inference on a single node with all CPU nodes.
+
+::
+
+   >>> torchrun --inference --throughput_mode script.py args
+
+2. Run single-instance inference on a single CPU node.
+
+::
+
+   >>> torchrun --inference --node_id 1 script.py args
+
+*** Multi-instance inference ***
+
+1. Multi-instance
+   By default, one instance per node. if you want to set the instance numbers and core per instance,
```

Comment:
`By default this tool runs one process per node`

Reply:
Done. Thanks for the correction.